index.bs

```diff
@@ -13110,12 +13123,14 @@ The following typographic conventions are used in this document:
 The following conventions are used in the algorithms in this document:
 
 *   Algorithms use the [=conventions of the ECMAScript specification=],
-    including the ! and ? notation for unwrapping completion records.
-*   Algorithms sometimes treat returning/throwing values and returning completion records
+    including the ! and ? notation for unwrapping [=Completion Records=].
```

Comment:
The link does go to Completion Record directly: https://tc39.github.io/ecma262/#sec-completion-record-specification-type